dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1328,8 +1328,9 @@ class WebCompileTest {
         '--no-pub',
       ]);
       watch?.stop();
-      final String outputFileName = path.join(directory, 'build/web/main.dart.js');
-      metrics.addAll(await getSize(outputFileName, metric: metric));
+      final String buildDir = path.join(directory, 'build', 'web');
+      final String outputFileName = path.join(buildDir, 'main.dart.js');
+      metrics.addAll(await getSize(buildDir, outputFileName, metric: metric));
```

Comment:
In addition to main.dart.js, I think we'd like to also track:

* build/canvaskit/canvaskit.js (not sure if this one varies in size much)
* build/canvaskit/canvaskit.wasm
* build/flutter.js (this should stay constant or grow, as we add features to it, for example dart2wasm bootstrap)

/cc @mdebbar more ideas?

Reply:
Those are good 👍 

I can't think of other ones right now, but we can always add more files later.